src/sql/workbench/contrib/jobManagement/browser/jobsView.component.ts

```diff
@@ -79,12 +79,12 @@ export class JobsViewComponent extends JobManagementView implements OnInit, OnDe
 
 	private _jobCacheObject: JobCacheObject;
 	private rowDetail: RowDetailView<IItem>;
-	private filterPlugin: HeaderFilter<IItem>;
-	private dataView: Slick.Data.DataView<IItem>;
+	private filterPlugin: any;
```

Comment:
You should be able to leave most of these typings - the filter maps are probably the only ones that will need the full set of fixes. But